aten/src/ATen/core/type_ptr.h

```diff
@@ -38,7 +38,7 @@ class SingletonTypePtr {
   }
 
  private:
-  T* repr_;
+  T* repr_{};
```

Comment:
Hmm, is this really necessary? Is  this class supposed to be called with default constructor?

Reply:
It's probably just cpp-core guidelines complaining about access a pointer that is not explicitly initialized as null.